docs/hub/spaces-sdks-gradio.md

```diff
@@ -55,3 +55,13 @@ This Python script uses a [🤗 Transformers pipeline](https://huggingface.co/do
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-hot-dog-gradio.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-hot-dog-gradio-dark.png"/>
 </div>
+
+## Embed Gradio Spaces on other webpages
+
+You can use the HTML `<iframe>` tag to embed a Gradio Space as an inline frame on other webpages. Simply include the name of your Space in the URL, delimited with a dash `"-"` between the user/org and Space name, followed by the `.hf.space` suffix. For example, the demo above can be embedded in these docs with the following tag:
+
+```
+<iframe src="https://NimaBoscarino-hotdog-gradio.hf.space" title="My awesome Gradio Space"></iframe>
```

Comment:
It would be cool if this URL could be copy-pasted directly from the Space at some point (similar to how one copies the org/repo ID)

Reply:
it will be done shortly